jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -23,42 +20,59 @@ class WfsCombineStep(Step):
         blur_size = integer(default=10)
         n_size = integer(default=2)
         suffix = string(default="wfscmb")
-    """ # noqa: E501
+    """  # noqa: E501
+
+    def make_output_path(self, basepath, *args, **kwargs):  # noqa:  ARG002
+        """
+        Ensure to bypass all stpipe filename formatting.
```

Comment:
```suggestion
        Bypass all stpipe filename formatting.
```
Does the checker not like using bypass as a verb?

Reply:
that's exactly what happened :P